src/pages/news/2021-03-30-oplossen-kwetsbaarheden-podcast.md

```diff
@@ -0,0 +1,20 @@
+---
+path: "/news/2021-03-30-2021-03-30-oplossen-kwetsbaarheden-podcast/"
+date: "2021-03-30"
+title: "Podcast interview: Kwetsbaarheden in OpenZaak oplossen"
+lang: nl
+template: default
+---
+
+## Podcast interview: Kwetsbaarheden in OpenZaak oplossen
+
+Enkele weken geleden was OpenZaak te zien in de livecast serie ['Let's talk about Public Code'](https://podcast.publiccode.net/) van de [Foundation for Public Code](publiccode.net). Het doel van deze show is om ervaringen te delen van codebases die werken met Public Code vanuit verschillende partijen, (ontwikkelaars, verkopers, ambtenaren, managers, etc.).
+
+De aflevering gewijd aan OpenZaak was gericht op het perspectief van de ontwikkelaars met betrekking tot een [kwetsbaarheid](https://github.com/publiccodenet/projects/issues/20) die werd ontdekt en opgelost binnen de gemeenschap en welke lessen daaruit werden getrokken.
+
+Het interview met [Sergei Maertens](https://github.com/sergei-maertens), technical lead bij [Maykin Media](https://github.com/maykinmedia) en [Tjerk Vaags](https://github.com/TjerkVaags), ontwikkelaar bij [Contezza](https://contezza.nl/), is beschikbaar in verschillende formaten:
+
+- Bekijk het op [Youtube](https://www.youtube.com/watch?v=1xojrumKgfA)
+- [Periscope](https://www.pscp.tv/w/1gqxvoRWjgwKB.]
```

Comment:
Typo : "- [Periscope](https://www.pscp.tv/w/1gqxvoRWjgwKB.] should end with ')' instead of ']'